jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -2313,6 +2317,19 @@ ClassDefiner makeHiddenClassDefiner(String name, byte[] template) {
 			ClassDefiner definer = new ClassDefiner(name, template, this);
 			return definer;
 		}
+		
+		public Class<?> ensureInitialized(Class<?> cls) throws IllegalArgumentException, IllegalAccessException {
+			if (cls.isArray()) {
+				throw new IllegalArgumentException(com.ibm.oti.util.Msg.getString("K0683", cls));
```

Comment:
The external messages need to be declared via preprocessor MSG on the line above, otherwise the messages won't be in the build.  The ExternalMessages-MasterIndex.properties file is just for tracking the indexes, it doesn't go into the build.